nipype/interfaces/base.py

```diff
@@ -1619,21 +1629,6 @@ def raise_exception(self, runtime):
     def _get_environ(self):
         return getattr(self.inputs, 'environ', {})
 
-    def version_from_command(self, flag='-v'):
-        cmdname = self.cmd.split()[0]
-        env = dict(os.environ)
-        if _exists_in_path(cmdname, env):
-            out_environ = self._get_environ()
-            env.update(out_environ)
-            proc = sp.Popen(' '.join((cmdname, flag)),
-                            shell=True,
-                            env=env,
-                            stdout=sp.PIPE,
-                            stderr=sp.PIPE,
-                            )
-            o, e = proc.communicate()
-            return o
```

Comment:
This is an API change. Even if not used internally, someone may depend on this method. Does nipype have a process for deprecating?

Reply:
Maybe we should start using https://pypi.python.org/pypi/deprecation ?